tests/src/ExistingSiteJavascript/SearchTest.php

```diff
@@ -62,6 +62,7 @@ protected function setUp(): void {
         'title' => 'Test unpublished ' . $node_type_label,
         'moderation_state' => 'draft',
       ]);
+      \Drupal::getContainer()->get('search_api.post_request_indexing')->destruct();
```

Comment:
Can we use `$this->container->get('search_api.post_request_indexing')->destruct();`

Reply:
Same as above